README.md

```diff
@@ -88,66 +111,45 @@ Working:
 - Configuration
   - Only Functional tests at this time.
   - Custom provider configurations are supported.
+- Compatibility with existing Akka.Persistence plugins is implemented via `table-mapping` setting.
 
 ## Incomplete
 
 - Tests for Schema Usage
-- Some Akka.NET specfic Journal Queries (those not mentioned above)
 - Cleanup of Configuration classes/fallbacks.
   - Should still be usable in most common scenarios including multiple configuration instances: see [`SqlServerCustomConfigSpec`](src/Akka.Persistence.Sql.Tests/SqlServer/SQLServerJournalCustomConfigSpec.cs) for test and examples.
 
-DB Compatibility:
-
-- SQL Server: Tests Pass
-- MS SQLite: Tests Pass
-- System.Data.SQLite: Functional tests pass, perf tests partially fail.
-  - For whatever reason SDS doesn't cope well here.
-- PostgreSQL: Tests pass
-- MySql: Not Tested Yet
-- Firebird: Not Tested Yet
-
-Compatibility with existing Providers is partially implemented via `table-compatibility-mode` flag. Please note there are not tests for compatibility with Sql.Common Query journals at this time:
-
-- SQL Server: Basic Persist and Recovery tests pass for both Snapshot and Journal.
-  - Still needs Delete tests
-- SQLite: Persist and Recovery tests pass for both Snapshot and Journal.
-- PostgreSQL: Basic Persist and Recovery tests pass for both Snapshot and Journal.
-  - Only Binary payloads are supported at this time.
-  - Note that not all possible permutations of table names have been tested.
-    - i.e. there may be concerns around casing
-- MySql: Not Implemented yet
+## Performance
 
-# Performance
-
-Tests based on i-7 8750H, 32GB Ram, 2TB SSD, Windows 10 version Version 10.0.19041.630.
+Tests based on AMD Ryzen 9 3900X, 32GB Ram, Windows 10 Version 22H2.
 Databases running on Docker WSL2.
 
 All numbers are in msg/sec.
 
-|Test |SqlServer (normal)       |  SqlServer Batching     | Linq2Db     |vs Normal| vs Batching|
-|:-------------  |:------------- | :----------: | -----------: | -----------: | -----------: |
-|Persist |  164|427| 235|143.29%|55.04%|
-|PersistAll |  782|875| 5609|717.26%|641.03%|
-|PersistAsync |  630|846| 16099|2555.40%|1902.96%|
-|PersistAllAsync |  2095|902| 15681|748.50%|1738.47%|
-|PersistGroup10 |  590|680| 1069|181.19%|157.21%|
-|PersistGroup100 |  607|965| 5537|912.19%|573.78%|
-|PersistGroup200 |  628|1356| 7966|1268.47%|587.46%|
-|PersistGroup25 |  629|675| 2189|348.01%|324.30%|
-|PersistGroup400 |  612|1011| 7237|1182.52%|715.83%|
-|PersistGroup50 |  612|654| 3867|631.86%|591.28%|
-|Recovering |  41903|38766| 42592|101.64%|109.87%|
-|Recovering8 |  75466|65515| 63960|84.75%|97.63%|
-|RecoveringFour |  59259|51355| 58437|98.61%|113.79%|
-|RecoveringTwo |  41745|35512| 41108|98.47%|115.76%|
+| Test            |          SqlServer | SqlServer Batching | Linq2Db | vs Normal | vs Batching |
```

Comment:
The new performance numbers look great - what's responsible for those improvements? Akka.NET v1.5, .NET 7, or a combination of factors?

Reply:
I really have no idea, all I did was re-run the benchmark on the latest SqlServer and Linq2Db version